src/Elewant/AppBundle/Entity/CustomType/Breed.php

```diff
@@ -41,7 +44,18 @@ public function convertToPHPValue($value, AbstractPlatform $platform)
      */
     public function convertToDatabaseValue($value, AbstractPlatform $platform)
     {
+        if ($value === null) {
```

Comment:
Maybe this was my mistake... I think this method should throw an Exception when used. This is a read model, and should never be converted back to a DB value at all. But if it where, I think it should never be null.